Integrations/integration-Wildfire.yml

```diff
@@ -254,10 +254,35 @@ script:
     description: Retrieve results for a file hash using WildFire
   - name: file
     description: Retrieve results for a file hash using WildFire
-    deprecated: true
```

Comment:
I believe it was deprecated so users will use the specific `wildfire-report` command. Didn't we decide to stop the generic `file`, `ip`...?